tests/runner/check_sender_receiver.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ducktape.cluster.localhost import LocalhostCluster
+from tests.ducktape_mock import test_context, session_context
+
+import logging
+import pytest
+from ducktape.tests.runner_client import Sender
+from ducktape.tests.runner import Receiver
+from ducktape.tests.event import ClientEventFactory, EventResponseFactory
+
+
+import multiprocessing as mp
+import os
+
+
+class CheckSenderReceiver(object):
+    def ready_response(self, client_id):
+        sender_event_factory = ClientEventFactory("test_1", 0, client_id)
+        sender = Sender(server_host='localhost', server_port=8888,
```

Comment:
Using fixed ports can cause problems for tests. The port won't always be available. I can't remember how we're setup on Jenkins wrt docker networking, but anywhere there might be concurrent tests running (including in the container via parallel pytest), this could cause flakiness.

Usually if you're doing a real test rather than mocking out some pieces, you want the code to support accepting `0` as the port, then have an accessor to get the bound port. That way it'll just choose randomly and the only way you run into issues is if you just run out of ports entirely.

Reply:
sounds good, Ill use the same port range that are used for the test runner.